package/kernel/linux/modules/netfilter.mk

```diff
@@ -39,6 +39,16 @@ endef
 
 $(eval $(call KernelPackage,nf-reject6))
 
+define KernelPackage/nf-conncount
+  SUBMENU:=$(NF_MENU)
+  TITLE:=Netfilter conncount support
+  KCONFIG:=$(KCONFIG_NF_CONNCOUNT)
+  DEPENDS:=+kmod-nf-conntrack
```

Comment:
Please add `HIDDEN:=1` here.
The `CONFIG_NETFILTER_CONNCOUNT` configuration option has no title and can not directly get selected, see: https://github.com/torvalds/linux/blob/master/net/netfilter/Kconfig#L108
With `HIDDEN:=1` it is also not possible to select it directly in OpenWrt which would then match the behavior in the kernel. This prevents someone selecting this in OpenWrt, but can not select the option in kernel kconfig.